### ISSUE TITLE
[BUG] failed to run `cargo run --release --example loop`

### ISSUE THREAD

**Issue by caiyesd** [risc0/risc0#1208]
## Bug Report

failed to run `cargo run --release --example loop`

## Steps to Reproduce
1.  install rustup

2.  install risczero
cargo install cargo-binstall
cargo binstall cargo-risczero
cargo risczero install

3. ensure risc0 toolchain is installed
```
(base) mint@Z400:~/risc0$ rustup toolchain list --verbose | grep risc0
risc0 (default) /home/mint/.local/share/cargo-risczero/toolchains/x86_64-unknown-linux-gnu_test-release-2/rust
```

4. download risc0 code and run  `cargo run --release --example loop`
```
git clone https://github.com/risc0/risc0
cd risc0
cargo run --release --example loop
```

## Expected behavior
The command can run successfully

## Your Environment
- risc0-zkvm version: v0.19.0 (I also tried v0.19.1 and main, the same issue)
- Rust version:  rustc 1.74.1 (a28077b28 2023-12-04)
- Platform/OS: ubuntu 22.04

## Additional context
build log:
```
(base) mint@Z400:~/risc0$ cargo run --release --example loop
   Compiling proc-macro2 v1.0.70
   Compiling unicode-ident v1.0.12
   Compiling cfg-if v1.0.0
   Compiling libc v0.2.150
   Compiling quote v1.0.33
   Compiling syn v2.0.39
   Compiling autocfg v1.1.0
   Compiling jobserver v0.1.27
   Compiling cc v1.0.83
   Compiling version_check v0.9.4
   Compiling bitflags v2.4.1
   Compiling serde v1.0.193
   Compiling memchr v2.6.4
   Compiling typenum v1.17.0
   Compiling pin-project-lite v0.2.13
   Compiling const-oid v0.9.5
   Compiling pkg-config v0.3.27
   Compiling generic-array v0.14.7
   Compiling rustix v0.38.27
   Compiling linux-raw-sys v0.4.12
   Compiling hashbrown v0.14.3
   Compiling crypto-common v0.1.6
   Compiling block-buffer v0.10.4
   Compiling subtle v2.5.0
   Compiling digest v0.10.7
   Compiling equivalent v1.0.1
   Compiling itoa v1.0.9
   Compiling futures-core v0.3.29
   Compiling once_cell v1.19.0
   Compiling vcpkg v0.2.15
   Compiling cpufeatures v0.2.11
   Compiling fastrand v2.0.1
   Compiling tempfile v3.8.1
   Compiling sha2 v0.10.8
   Compiling futures-sink v0.3.29
   Compiling serde_derive v1.0.193
   Compiling tracing-attributes v0.1.27
   Compiling slab v0.4.9
   Compiling futures-task v0.3.29
   Compiling futures-channel v0.3.29
   Compiling hex v0.4.3
   Compiling value-bag v1.4.2
   Compiling log v0.4.20
   Compiling futures-macro v0.3.29
   Compiling pin-utils v0.1.0
   Compiling option-ext v0.2.0
   Compiling dirs-sys v0.4.1
   Compiling zeroize_derive v1.4.2
   Compiling getrandom v0.2.11
   Compiling either v1.9.0
   Compiling bytes v1.5.0
   Compiling rand_core v0.6.4
   Compiling directories v5.0.1
   Compiling aho-corasick v1.1.2
   Compiling zeroize v1.7.0
   Compiling regex-syntax v0.8.2
   Compiling fnv v1.0.7
   Compiling ppv-lite86 v0.2.17
   Compiling risc0-build-kernel v0.19.0 (/home/mint/risc0/risc0/build_kernel)
   Compiling num-traits v0.2.17
   Compiling ryu v1.0.15
   Compiling foreign-types-shared v0.1.1
   Compiling foreign-types v0.3.2
   Compiling tracing-core v0.1.32
   Compiling openssl-macros v0.1.1
   Compiling native-tls v0.2.11
   Compiling tracing v0.1.40
   Compiling bytemuck_derive v1.5.0
   Compiling tinyvec_macros v0.1.1
   Compiling scopeguard v1.2.0
   Compiling smallvec v1.11.2
   Compiling anyhow v1.0.75
   Compiling tinyvec v1.6.0
   Compiling indexmap v2.1.0
   Compiling openssl-sys v0.9.97
   Compiling openssl-probe v0.1.5
   Compiling httparse v1.8.0
   Compiling memoffset v0.9.0
   Compiling crossbeam-utils v0.8.16
   Compiling futures-util v0.3.29
   Compiling paste v1.0.14
   Compiling thiserror v1.0.50
   Compiling try-lock v0.2.5
   Compiling crc32fast v1.3.2
   Compiling want v0.3.1
   Compiling rand_chacha v0.3.1
   Compiling bytemuck v1.14.0
   Compiling thiserror-impl v1.0.50
   Compiling num_cpus v1.16.0
   Compiling openssl-src v300.2.0+3.2.0
   Compiling crossbeam-epoch v0.9.15
   Compiling mio v0.8.10
   Compiling socket2 v0.5.5
   Compiling percent-encoding v2.3.1
   Compiling tower-service v0.3.2
   Compiling thiserror-core v1.0.50
   Compiling unicode-bidi v0.3.14
   Compiling futures-io v0.3.29
   Compiling httpdate v1.0.3
   Compiling byteorder v1.5.0
   Compiling form_urlencoded v1.2.1
   Compiling tokio v1.34.0
   Compiling rand v0.8.5
   Compiling thiserror-core-impl v1.0.50
   Compiling num-integer v0.1.45
   Compiling static_assertions v1.1.0
   Compiling adler v1.0.2
   Compiling openssl v0.10.61
   Compiling syn v1.0.109
   Compiling glob v0.3.1
   Compiling miniz_oxide v0.7.1
   Compiling twox-hash v1.6.3
   Compiling http v0.2.11
   Compiling regex-automata v0.4.3
   Compiling num-bigint v0.3.3
   Compiling lazy_static v1.4.0
   Compiling rayon-core v1.12.0
   Compiling cpp_demangle v0.4.3
   Compiling regex v1.10.2
   Compiling ruzstd v0.4.0
   Compiling crossbeam-deque v0.8.3
   Compiling flate2 v1.0.28
   Compiling tokio-util v0.7.10
   Compiling tokio-macros v2.2.0
   Compiling signal-hook-registry v1.4.1
   Compiling stable_deref_trait v1.2.0
   Compiling mime v0.3.17
   Compiling fallible-iterator v0.3.0
   Compiling radium v0.7.0
   Compiling gimli v0.28.1
   Compiling h2 v0.3.22
   Compiling object v0.32.1
   Compiling http-body v0.4.5
   Compiling serde_urlencoded v0.7.1
   Compiling unicode-normalization v0.1.22
   Compiling risc0-sys v0.19.0 (/home/mint/risc0/risc0/sys)
   Compiling itertools v0.11.0
   Compiling zstd-sys v2.0.9+zstd.1.5.5
   Compiling memmap2 v0.5.10
   Compiling unicase v2.7.0
   Compiling backtrace v0.3.69
   Compiling matrixmultiply v0.3.8
   Compiling socket2 v0.4.10
   Compiling encoding_rs v0.8.33
   Compiling ipnet v2.9.0
   Compiling base64 v0.21.5
   Compiling rustc-demangle v0.1.23
   Compiling tap v1.0.1
   Compiling wyz v0.5.1
   Compiling addr2line v0.21.0
   Compiling hyper v0.14.27
   Compiling prost-derive v0.12.3
   Compiling idna v0.5.0
   Compiling rayon v1.8.0
   Compiling addchain v0.2.0
   Compiling tokio-native-tls v0.3.1
   Compiling risc0-core v0.19.0 (/home/mint/risc0/risc0/core)
   Compiling heck v0.4.1
   Compiling rawpointer v0.2.1
   Compiling semver v1.0.20
   Compiling funty v2.0.0
   Compiling serde_json v1.0.108
   Compiling bitvec v1.0.1
   Compiling hyper-tls v0.5.0
   Compiling ff_derive v0.13.0
   Compiling url v2.5.0
   Compiling futures-executor v0.3.29
   Compiling num-complex v0.4.4
   Compiling autotools v0.2.6
   Compiling risc0-zkp v0.19.0 (/home/mint/risc0/risc0/zkp)
   Compiling camino v1.1.6
   Compiling prettyplease v0.2.15
   Compiling zstd-safe v5.0.2+zstd.1.5.2
   Compiling protobuf-src v1.1.0+21.5
   Compiling ndarray v0.15.6
   Compiling futures v0.3.29
   Compiling mime_guess v2.0.4
   Compiling reqwest v0.11.22
   Compiling ff v0.13.0
   Compiling prost v0.12.3
   Compiling risc0-circuit-recursion-sys v0.19.0 (/home/mint/risc0/risc0/circuit/recursion-sys)
   Compiling risc0-circuit-rv32im-sys v0.19.0 (/home/mint/risc0/risc0/circuit/rv32im-sys)
   Compiling blake2 v0.10.6
   Compiling home v0.5.5
   Compiling risc0-zkvm-platform v0.19.0 (/home/mint/risc0/risc0/zkvm/platform)
   Compiling termcolor v1.4.0
   Compiling fixedbitset v0.4.2
   Compiling humantime v2.1.0
   Compiling petgraph v0.6.4
   Compiling which v4.4.2
   Compiling prost-types v0.12.3
   Compiling downloader v0.2.7
   Compiling cargo-platform v0.1.5
   Compiling lock_api v0.4.11
   Compiling elf v0.7.4
   Compiling parking_lot_core v0.9.9
   Compiling utf8parse v0.2.1
   Compiling multimap v0.8.3
   Compiling regex-syntax v0.6.29
   Compiling regex-automata v0.1.10
   Compiling prost-build v0.12.3
   Compiling anstyle-parse v0.2.3
   Compiling risc0-binfmt v0.19.0 (/home/mint/risc0/risc0/binfmt)
   Compiling cargo_metadata v0.17.0
   Compiling risc0-circuit-recursion v0.19.0 (/home/mint/risc0/risc0/circuit/recursion)
   Compiling scroll_derive v0.11.1
   Compiling is-terminal v0.4.9
   Compiling overload v0.1.1
   Compiling colorchoice v1.0.0
   Compiling anstyle v1.0.4
   Compiling risc0-circuit-rv32im v0.19.0 (/home/mint/risc0/risc0/circuit/rv32im)
   Compiling anstyle-query v1.0.1
   Compiling docker-generate v0.1.3
   Compiling risc0-build v0.19.0 (/home/mint/risc0/risc0/build)
   Compiling anstream v0.6.4
   Compiling nu-ansi-term v0.46.0
   Compiling env_logger v0.10.1
   Compiling scroll v0.11.0
   Compiling matchers v0.1.0
   Compiling lazy-regex-proc_macros v2.4.1
   Compiling sharded-slab v0.1.7
   Compiling tracing-log v0.2.0
   Compiling erased-serde v0.3.31
   Compiling typetag-impl v0.2.13
   Compiling thread_local v1.1.7
   Compiling inventory v0.3.13
   Compiling strsim v0.10.0
   Compiling plotters-backend v0.3.5
   Compiling ciborium-io v0.2.1
   Compiling half v1.8.2
   Compiling downcast-rs v1.2.0
   Compiling plain v0.2.3
   Compiling clap_lex v0.6.0
   Compiling clap_builder v4.4.11
   Compiling goblin v0.7.1
   Compiling rrs-lib v0.1.0
   Compiling ciborium-ll v0.2.1
   Compiling plotters-svg v0.3.5
   Compiling typetag v0.2.13
   Compiling tracing-subscriber v0.3.18
   Compiling lazy-regex v2.5.0
   Compiling bonsai-sdk v0.5.0 (/home/mint/risc0/bonsai/sdk)
   Compiling risc0-zkvm-methods v0.19.0 (/home/mint/risc0/risc0/zkvm/methods)
   Compiling clap_derive v4.4.7
   Compiling crypto-bigint v0.5.5
   Compiling itertools v0.10.5
   Compiling bincode v1.3.3
   Compiling num-derive v0.4.1
   Compiling cast v0.3.0
   Compiling same-file v1.0.6
   Compiling walkdir v2.4.0
   Compiling criterion-plot v0.5.0
   Compiling clap v4.4.11
risc0-zkvm-methods-guest: Starting build for riscv32im-risc0-zkvm-elfm-sys(build), risc0-circuit-recursion-sys(build), protobuf-src(build), risc0-zkvm-methods(build), zstd-sys(build)
risc0-zkvm-methods-guest: error: could not execute process ` -vV` (never executed)
risc0-zkvm-methods-guest:
risc0-zkvm-methods-guest: Caused by:
risc0-zkvm-methods-guest:   No such file or directory (os error 2)
error: failed to run custom build command for `risc0-zkvm-methods v0.19.0 (/home/mint/risc0/risc0/zkvm/methods)`

Caused by:
  process didn't exit successfully: `/home/mint/risc0/target/release/build/risc0-zkvm-methods-58b921e67529646c/build-script-build` (exit status: 101)
  --- stdout
  Building guest package risc0-zkvm-methods.risc0-zkvm-methods-guest
  cargo:rerun-if-env-changed=RISC0_SKIP_BUILD
  Using rustc:
  cargo:rerun-if-env-changed=RISC0_RUST_SRC
  Building guest package: cargo build --target riscv32im-risc0-zkvm-elf
  cargo:rerun-if-env-changed=RISC0_BUILD_DEBUG
warning: build failed, waiting for other jobs to finish...
warning: cxx/step_verify_bytes.cpp: In function ‘risc0::Fp risc0::circuit::recursion::step_verify_bytes(void*, void (*)(void*, const char*, const char*, const risc0::Fp*, size_t, risc0::Fp*, size_t), size_t, size_t, risc0::Fp**)’:
warning: cxx/step_verify_bytes.cpp:33:10: warning: unused variable ‘mask’ [-Wunused-variable]
warning:    33 |   size_t mask = steps - 1;
warning:       |          ^~~~
warning: cxx/step_verify_bytes.cpp:32:28: warning: unused parameter ‘ctx’ [-Wunused-parameter]
warning:    32 | Fp step_verify_bytes(void* ctx, HostBridge host, size_t steps, size_t cycle, Fp** args) {
warning:       |                      ~~~~~~^~~
warning: cxx/step_verify_bytes.cpp:32:44: warning: unused parameter ‘host’ [-Wunused-parameter]
warning:    32 | Fp step_verify_bytes(void* ctx, HostBridge host, size_t steps, size_t cycle, Fp** args) {
warning:       |                                 ~~~~~~~~~~~^~~~
warning: cxx/step_verify_bytes.cpp:32:71: warning: unused parameter ‘cycle’ [-Wunused-parameter]
warning:    32 | Fp step_verify_bytes(void* ctx, HostBridge host, size_t steps, size_t cycle, Fp** args) {
warning:       |                                                                ~~~~~~~^~~~~
warning: cxx/step_verify_bytes.cpp:32:83: warning: unused parameter ‘args’ [-Wunused-parameter]
warning:    32 | Fp step_verify_bytes(void* ctx, HostBridge host, size_t steps, size_t cycle, Fp** args) {
warning:       |          
```

**Comment by flaub** [comment]
This is likely fixed with:
```
git lfs install
git lfs pull
```

**Comment by flaub** [comment]
I think this has been resolved in 0.20, re-open if this is still failing.
